atlas-content-modeler/tutorial/models/fields/text/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+slug: /content-engine/tutorial/models/fields/text/introduction
+title: Content Engine Fields - Text
+description: Learn how to use Text Fields in Content Engine Models.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../introduction.mdx).
+
+## Text Field
+
+The text field is used for any type of regular text that doesn't include HTML or markdown. You can choose single or multiple lines (textarea) as well as minimum and maximum characer count in advanced settings.
+
+### Steps to Use the Text Field
+---
+1. After navigating to a model, click the + button or if it's the first field, click on the Text option
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.04.05_PM.png)
+
+1. Choose the Text Field type
+
+1. Fill in the form fields to fit your needs
+
+    ![Content Engine sync](../../../../images/Screen_Shot_2021-09-01_at_12.04.17_PM.png)
+
+1. To make this field the entry title for the model, check the "Use this field as the entry title" box under Title Field
+
+1. Choose Single line for shorter text and choose Multiple lines for a textarea and longer text
+
+1. Click on Advanced Settings to choose a minimum and maximum character length
+
+1. Click Create when done to save the field to the model
+
+1. Click Cancel to stop field creation
+
+## Feedback
+---
+- **Questions?** Email us ce-beta@wpengine.com
```

Comment:
Is ce-beta@wpengine.com the correct support email for Content Modeler?

Reply:
@matthewguywright same as https://github.com/wpengine/headless-docs/pull/71#discussion_r706343782